app/models/user.rb

```diff
@@ -15,9 +19,29 @@ class User < ApplicationRecord
     event :close do
       after do
         pp "close"
+        pp"検証"
       end
 
       transition all - [:closed] => :closed
     end
   end
-end
\ No newline at end of file
+
+  def taunt_police
+    pp "taunt police"
+  end
+
+  def taunt_police2
+    pp "taunt police"
+  end
+
+  def taunt_police3
+    pp "taunt police"
+  end
+
+  def taunt_police4
```

Comment:
```suggestion
    pp 'taunt police'
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>